backend/src/routes.js

```diff
@@ -1,12 +1,32 @@
-const express = require('express')
+const express = require('express');
+
 const router = express.Router();
 
+
 router.get('/users', (req, res) => {
     res.send(`<h1>No users yet :(</h1>`)
-})
+});
+
+
+//route to get a single user
+router.get('/users/:id', async (req,res) => {
+    try{
+        let user = await Users.findById(req.params.id);
+        if (user){
+            delete user.password;
+            res.status(200).json(user);
```

Comment:
Please wrap this in an object with a "user" property as per the acceptance criteria.